bin/juxtaposer/tester/db/mssql/mssql.go

```diff
@@ -0,0 +1,102 @@
+package mssql
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"net/url"
+	"strings"
+
+	_ "github.com/denisenkom/go-mssqldb"
+
+	"github.com/cyberark/secretless-broker/bin/juxtaposer/tester/api"
+	sql_db_tester "github.com/cyberark/secretless-broker/bin/juxtaposer/tester/db/sql"
+)
+
+// MssqlTester is a wrapping struct around the basic SQL tester
+type MssqlTester struct {
+	sql_db_tester.SqlDatabaseTester
+}
+
+// NewTester creates a new instance of the MSSQL DB tester
+func NewTester() (api.DbTester, error) {
+	tester := &MssqlTester{}
+
+	return tester, nil
+}
+
+// sqlserver://username:password@host:port?database=master&param2=value
+
+// GetQueryMarkers returns part of the query string that will be paramerized as it's
+// different between databases. In this case, the params are defined using `@p<num>`.
+func (tester *MssqlTester) GetQueryMarkers(length int) string {
+	markers := []string{}
+	for markerIndex := 1; markerIndex <= length; markerIndex++ {
+		markers = append(markers, fmt.Sprintf("@p%d", markerIndex))
+	}
+	return strings.Join(markers, ", ")
+}
+
+// Connect is used to initialize a testing connection to the SQL database
+func (tester *MssqlTester) Connect(options api.DbTesterOptions) error {
```

Comment:
Codeclimate doesn't like that there's six return statements here (it says maximum of four)

Reply:
Yeah CC is silly - it's practically impossible to get that check passing when you are also checking for errors so I would ignore it.